impeller/display_list/dl_dispatcher.cc

```diff
@@ -582,25 +579,26 @@ void DlDispatcherBase::drawCircle(const DlPoint& center, DlScalar radius) {
 }
 
 // |flutter::DlOpReceiver|
-void DlDispatcherBase::drawRRect(const SkRRect& rrect) {
+void DlDispatcherBase::drawRoundRect(const DlRoundRect& rrect) {
   AUTO_DEPTH_WATCHER(1u);
 
-  if (skia_conversions::IsNearlySimpleRRect(rrect)) {
-    GetCanvas().DrawRRect(skia_conversions::ToRect(rrect.rect()),
-                          skia_conversions::ToSize(rrect.getSimpleRadii()),
-                          paint_);
+  if (rrect.GetRadii().AreAllSame()) {
```

Comment:
This needs to use the  IsNearlySimpleRRect rrect that has a tolerance for floating point errors

Reply:
I changed the method to apply a (configurable) tolerance so it should behave as before wherever it is used.